conf/application.conf

```diff
@@ -197,6 +197,7 @@ external-url {
   contact-hmrc="https://www.gov.uk/government/organisations/hm-revenue-customs/contact/capital-gains-tax-enquiries-for-individuals-employees-and-self-employed"
   trust-registered="https://www.gov.uk/trusts-taxes/trustees-tax-responsibilities"
   personal-representative="https://www.gov.uk/government/publications/death-personal-representatives-and-legatees-hs282-self-assessment-helpsheet/hs282-death-personal-representatives-and-legatees-2018"
+  personal-representative-in-period-of-admin="https://www.gov.uk/government/publications/death-personal-representatives-and-legatees-hs282-self-assessment-helpsheet/hs282-death-personal-representatives-and-legatees-2018"
```

Comment:
do we still need the config just above this then? It looks the same

Reply:
I kept it separate for flexibility purposes and intent. But looking at this again, there should be only one entry

Reply:
removed this